packages/framework/fluid-static/src/types.ts

```diff
@@ -12,6 +12,13 @@ export type LoadableObjectRecord = Record<string, IFluidLoadable>;
 
 export type LoadableObjectClassRecord = Record<string, LoadableObjectClass<any>>;
 
+/**
+ * Construct a type with a set of properties K of type T
+ */
+export type LoadableObjectInstanceTypeRecord<K extends LoadableObjectClassRecord> = {
```

Comment:
this is both confusing, and not specific. at a glance i don't understand how to create one of these. it sees like it should be much simplier like
```
type AvailabeDds = ISharedMap | ISharedString
type Schema = Record<string, AvailabeDds>
```

Reply:
i also wonder if the default should be something much more useful like `{root: IsharedMap}` rather than LoadableObjectClassRecord which will produce a bunch of any's if they are not set.